politeiawww/codetracker/github/api/ratelimit.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+)
+
+const (
+	rateLimitURL = "https://api.github.com/rate_limit"
+)
+
+// RateLimit determines if the current rate limit for the client is about
+// to be tripped.  If so, it will go to sleep for a given perioud of time.
+func (a *Client) RateLimit() (RateLimitRule, error) {
+	defer a.rateLimitMtx.Unlock()
+	a.rateLimitMtx.Lock()
+
+	for {
+		if a.rateLimit.Remaining == 0 {
```

Comment:
`if rateLimit.Remaining != 0 { continue }` would unindent this and make it more obvious.